src/EntityManagerFactory.php

```diff
@@ -20,15 +21,10 @@ public function create($entityManagerName)
                 $isDevMode = true;
                 $paths = array(__DIR__ . "/Models");
 
-                $dbParams = array(
-                    'driver' => 'pdo_mysql',
-                    'user' => 'root',
-                    'password' => '',
-                    'dbname' => 'test',
-                );
-
+                $dbParams = Yaml::parse(file_get_contents(__DIR__ . '/../config/database.yml'));
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'create'.](https://www.codacy.com/app/Managlea/EntityManager/file/2313601654/issues/source?bid=3149133&fileBranchId=3199776#l24)